tests/db/migration_tests/dump_tool.py

```diff
@@ -157,33 +159,41 @@ async def test_dump_db(server, client, postgres_db, database_name):
     env_1_version += 1
     await wait_for_version(client, env_id_1, env_1_version)
 
-    await client.release_version(
-        env_id_1, env_1_version, push=True, agent_trigger_method=const.AgentTriggerMethod.push_full_deploy
+    check_result(
+        await client.release_version(
+            env_id_1, env_1_version, push=True, agent_trigger_method=const.AgentTriggerMethod.push_full_deploy
+        )
     )
 
     await _wait_until_deployment_finishes(client, env_id_1, env_1_version, 20)
 
     # a version that is release, but not deployed
-    await client.notify_change(id=env_id_1, update=False)
+    check_result(await client.notify_change(id=env_id_1, update=False))
 
     env_1_version += 1
     await wait_for_version(client, env_id_1, env_1_version)
-    await client.release_version(
-        env_id_1, env_1_version, push=False, agent_trigger_method=const.AgentTriggerMethod.push_full_deploy
+    check_result(
+        await client.release_version(
+            env_id_1, env_1_version, push=False, agent_trigger_method=const.AgentTriggerMethod.push_full_deploy
+        )
     )
 
     await populate_facts_and_parameters(client, env_id_1)
 
     await wait_for_version(client, env_id_1, env_1_version)
 
-    await client.release_version(
-        env_id_1, env_1_version, push=False, agent_trigger_method=const.AgentTriggerMethod.push_full_deploy
+    env_1_version += 1
```

Comment:
otherwise the next call fails as the version is already released.

Reply:
I think we have to move the `env_1_version += 1` up until before the `await wait_for_version(client, env_id_1, env_1_version)` call, no?

Reply:
When I do that it fails. But I don't think this wait_for_version is required

Reply:
Why does it fail?